src/tests/operators/math/cosh/cosh_fp_test.cairo

```diff
@@ -0,0 +1,49 @@
+use array::SpanTrait;
+use array::{ArrayTrait};
+use orion::operators::tensor::implementations::impl_tensor_fp;
+use orion::operators::tensor::core::{TensorTrait, ExtraParams};
+use orion::numbers::fixed_point::core::{FixedTrait, FixedType, FixedImpl};
+use orion::numbers::fixed_point::implementations::impl_8x23;
+use debug::PrintTrait;
+
+#[test]
+#[available_gas(2000000000)]
+fn cosh_fp_test() {
+    let mut sizes = ArrayTrait::new();
+    sizes.append(5);
+
+    let mut arr = ArrayTrait::<FixedType>::new();
+    arr.append(FixedTrait::new_unscaled(2, true));
+    arr.append(FixedTrait::new_unscaled(1, true));
+    arr.append(FixedTrait::new_unscaled(0, false));
+    arr.append(FixedTrait::new_unscaled(1, false));
+    arr.append(FixedTrait::new_unscaled(2, false));
+
+    let extra = ExtraParams { fixed_point: Option::Some(FixedImpl::FP8x23(())) };
+
+    let tensor = TensorTrait::<FixedType>::new(sizes.span(), arr.span(), Option::Some(extra));
+
+    let result = tensor.cosh().data;
+
+    // let result1 = *result.at(0);
```

Comment:
Why is this commented? If not needed / used better remove the code :)

Reply:
Ah I just forgot to remove the print() when I was debugging...I removed all print()'s and also removed the import for 'use debug::PrintTrait' in all of the tests that still had it.